21.py

```diff
@@ -0,0 +1,50 @@
+# https://projecteuler.net/problem=21
+import math
+
+
+def is_proper_divisor(divisor, n):
+    if divisor >= n:
+        return False
+    if n % divisor == 0:
+        return True
+    return False
+
+
+def sum_proper_divisors(n):
+    sum = 1
+
+    for x in range(2, math.floor(math.sqrt(n)) + 1):
```

Comment:
i guess this could just be math.ceil() instead of math.floor() + 1

Reply:
ya personally i did int(n**0.5+1)

Reply:
nice to know you can do this without bringing in the math library